docs/src/user-guide/guides-using-object-storage/archive-storage.md

```
@@ -0,0 +1,64 @@
+# Storing archives
+
+To store compressed archives on S3, you'll need to:
+
+1. Enable the CLP IAM user to access the S3 path where archives should be stored.
+2. Configure CLP to store archives under the relevant S3 path.
+
+## IAM user configuration
+
+Attach the following policy to the CLP IAM user by following [this guide][add-iam-policy].
```

Markdownlint (0.37.0)

Missing link or image reference definition: "add-iam-policy"
Reference links and images should use a label that is defined

(MD052, reference-links-images)